articles/overview/quantum-computers-and-simulators.md

```diff
@@ -0,0 +1,64 @@
+- Signals are sent to the qubits using microwaves or lasers.
+
+Quantum computers face a multitude of challenges to operate correctly.  Labs that hold quantum computers are designed to minimize vibrations, and quantum computers are often sealed in a vacuum chamber to isolate them.  Error correction in quantum computers is a significant issue, and scaling up (adding more qubits) increases the error rate.
+
+Because of these limitations, a quantum PC for your desktop is far in the future, but a commercially-viable lab-based quantum computer is closer.
+
+## Quantum simulators
+
+Quantum simulators that run on classical computers allow you to simulate the execution of quantum algorithms on a quantum system.  Microsoft’s Quantum Development Kit (QDK) includes a full-state vector simulator along with other specialized quantum simulators.
+
+## Topological qubit
+
+Microsoft is developing a quantum computer based on topological qubits. A topological qubit will be less impacted by changes in its environment, therefore reducing the degree of external error correction. Topological qubits feature increased stability and resistance to environmental noise, which means they can more readily scale and remain reliable longer.
+
+## Microsoft and quantum hardware partnerships
+
+Microsoft is partnering with quantum hardware manufacturers IonQ, Honeywell, and QCI to make quantum computers accessible to developers in the future. Leveraging the Azure Quantum platform, developers will be able to use Microsoft’s Quantum Development Kit (QDK) and Q# to write quantum programs and run them remotely.
+
+## Quantum computations
+
+Performing computations on a quantum computer or quantum simulator follow a basic process:
+
+- Access the qubits
+- Initialize the qubits to the desired state
+- Perform operations to transform the states of the qubits
+- Measure the new states of the qubits
+
+Initializing and transforming qubits is done using **quantum operations** (sometimes called quantum gates). Quantum operations are similar to logic operations in classical computing, such as AND, OR, NOT, and XOR. An operation can be as basic as flipping a qubit's state from 1 to 0 or entangling a pair of qubits, to using multiple operations in series to affect the probability of a superposed qubit collapsing one way or the other.
+
+Measuring the result of the computation tells us an answer, but not necessarily the correct answer. Because the result is based on the probability that was configured by the quantum operations, quantum computations are run multiple times to get a probability distribution and refine the accuracy of the results.  Assurance that an operation returned a correct answer is known as quantum verification and is a significant challenge in quantum computing.
```

Comment:
This is true of some kinds of quantum algorithms, but is not true in general. Deutsch–Jozsa succeeds with probability 100%, for instance, up to noise in the quantum device being used.